Description: In Debian, the clamav-daemon socket is at /run/clamav/clamd.ctl
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/configs/contentscanners/clamdscan.conf.in
+++ b/configs/contentscanners/clamdscan.conf.in
@@ -1,7 +1,7 @@
 plugname = 'clamdscan'
 
 # edit this to match the location of your ClamD UNIX domain socket
-#clamdudsfile = '/var/run/clamav/clamd.sock'
+clamdudsfile = '/run/clamav/clamd.ctl'
 
 # If this string is set, the text it contains shall be removed from the
 # beginning of filenames when passing them to ClamD.
